### PR TITLE
Add Jenkinsfile

### PR DESCRIPTION
**Pull request by cbeer**

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -64,7 +64,7 @@ end
 
 group :production, :dortest do
   gem 'mysql2'
-  gem 'activerecord-mysql-adapter'
+  # gem 'activerecord-mysql-adapter'
```
Comment (jcoyne) [inline Gemfile]: Remove?

File: Jenkinsfile [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+    deleteDir()
+  }
+
+  stage('Build') {
+    checkout scm
+
+    sh '''#!/bin/bash -l
+    rvm use 2.3.4@hydrus --create
+
+    git submodule update --init
```
Comment (jcoyne) [inline Jenkinsfile]: Do we still need this?

Comment (cbeer) [inline Jenkinsfile]: No idea, I was just following the README.

Comment (jcoyne) [inline Jenkinsfile]: You don't. See https://github.com/sul-dlss/hydrus/pull/94

Comment (cbeer) [inline Jenkinsfile]: 👍

**Pull request merged**